code/modules/mob/living/carbon/human/species_types/oozelings.dm

```diff
@@ -182,6 +182,19 @@
 		H.gib()
 		return
 
+/datum/species/oozeling/help(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+	. = ..()
+	if(. && target != user && user.zone_selected == BODY_ZONE_CHEST && target.on_fire)
```

Comment:
It should work on any bodypart selected imo, may be awkward to have to select chest every time

Reply:
fixed